server/programs/Cargo.toml

```diff
@@ -19,6 +19,7 @@ borsh = "*"
 borsh-derive = "*"
 bytemuck = "*"
 bytemuck_derive = "*"
+jupiter-cpi = { git = "https://github.com/jup-ag/jupiter-cpi", rev = "5eb8977" }
```

Comment:
The client currently only support crates.io crates, but it should be relatively easy to also support crates from `git`.